src/main/java/com/elearningbackend/entity/Category.java

```diff
@@ -81,7 +85,7 @@ public int hashCode() {
         return result;
     }
 
-    @OneToMany(mappedBy = "category", cascade = CascadeType.ALL)
+    @OneToMany(mappedBy = "category", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
đã đùn cascade rồi thì không dùng thằng fetch nữa

Reply:
CASCADE và FETCH k liên quan đến nhau nha. Đọc đây để biết thêm chi tiết
- https://stackoverflow.com/questions/31318788/cascadetype-vs-fetchtype

Reply:
anh đang nói là sử dụng fecth thì nó thường gây lỗi với việc truyền tải dữ liệu hoặc nó không làm việc https://stackoverflow.com/questions/42328532/why-lazy-fetching-is-not-working-jpa

Reply:
Thế anh phải nói rõ chứ bảo dùng cascade rồi k dùng thằng fetch nữa thì e thấy k liên quan. Vì 2 thằng này chức năng k bị chồng chéo nhau.

Còn fetch lazy loading nó k hoạt động đc thật. Vì em đang k muốn nó lấy dữ liệu của các entity có quan hệ với nó (category) như subcategory + entity quan hệ với subcategory (như questionBank). A có giải pháp nào ko? Hiện tại thì em có thể bỏ thằng này đi, vì em đã có giải pháp ăn xổi khác rồi. Đó là dùng Json Ignore để ignore trên dto. Tuy nhiên, về cơ bản thì repository nó vẫn lấy hết :/

Reply:
anh nhớ là anh có hỏi thầy Duy 1 lần rồi: " nếu đứng ở vij trí thằng 1 thì không quan tâm thằng n nó có gì nên mình có thể xóa bỏ luôn" anh nhớ là vậy!

Reply:
hoặc dùng thằng này : https://stackoverflow.com/questions/31319358/jsonmanagedreference-vs-jsonbackreference

Reply:
Anh đọc hết class đi đã. Cái thằng managed reference và back reference k liên quan đến vấn đề này. Em có add vào ngay từ đẩu r mà